workshop/README.md

```diff
@@ -25,43 +41,47 @@ The invoice transaction accepts three parameters (send by the freelancer to the
+Let's explore the different functions and their implementations in the `invoice_transaction.js` file. Open the file so you can follow along with the code.<br><br>
 
 #### Getter for TYPE
-Static function that returns {number}.
+Static function that returns **{number}**. This is the type identifier for the transaction. We use this number to uniquely register our new custom transaction in our blockchain network. This means that we can't register transactions with the same transaction TYPE.
+
+**Important: The Lisk Protocol reserves transaction [types 0 - 4](https://lisk.io/documentation/lisk-protocol/transactions) and Lisk Core reserves transaction types 5 - 7.**<br><br>
 
 #### Getter for FEE
-Static function that returns {number}. We are converting a fee to Beddows which is the lowest denominal in the Lisk ecosystem: 10 ** 8 = 1 LSK (100000000).
+Static function that returns **{string}**. The fee must be paid when sending this transaction to the blockchain network. We are converting a fee from LSK to Beddows which is the lowest denominal in the Lisk ecosystem: `10 ** 8 = 1 LSK (100000000)`.
+
+_Note: It is possible to use a zero fee._<br><br>
 
 #### prepare()
-As we will be only modifying the sender (freelancer) their account, we just need to store this account in the cache of the key-value store. At the moment, we are calling the method through `super`.
+The prepare function is used to cache data from the database in a key-value store. This data can later be accessed in the `applyAsset` or `undoAsset` functions to update the state of an account.
 
-However, this is a bad practice as the implementation might change inside the `BaseTransaction`. Therefore, we want to open the `BaseTransaction` to copy the code that caches the sender account.
+As we will be only modifying the sender (freelancer) his account, we just need to cache this account in the key-value store. At the moment, we are implicitly caching the account by calling the implementation in the BaseTransaction class through `super`.
 
-**Task 1: Go to [BaseTransaction class](https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-transactions/src/base_transaction.ts) at line 399 to copy the implementation and paste it in the prepare() function for InvoiceTransaction replacing the `super` call.**
+However, this is a bad practice as the implementation might change inside the `BaseTransaction`. Therefore, we want to open the `BaseTransaction` to copy the code that caches the sender account.<br><br>
 
-_The solution for each step can be found in the `Solution: Invoice Transaction` section._
+**Task 1: Go to [BaseTransaction class](https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-transactions/src/base_transaction.ts#L400-L404) and copy the highlighted implementation of the `prepare()` function (don't copy the whole function definition as it's created in TypeScript) and paste it in the `prepare()` function for InvoiceTransaction replacing the `super` call.**
```

Comment:
The highlighted code snippet seems wrong to me. Doesn't fit to the solution as well.

Reply:
You are right, weird, I've updated it to be the right selection of lines thanks for catching!